WordPress/Classes/Services/SignupService.swift

```diff
@@ -3,146 +3,15 @@ import CocoaLumberjack
 import WordPressShared
 
 
```

Comment:
This file has barely any contents now. Should it be renamed? It's less `SignupService` and more `GoogleSignupService` 😜

Reply:
Ahhhh good point sir!! Let's rename in another PR!!.